target/linux/kirkwood/patches-4.14/110-pogo_e02.patch

```diff
@@ -30,3 +30,43 @@
  			label = "pogo_e02:orange:fault";
  			gpios = <&gpio1 17 GPIO_ACTIVE_LOW>;
  		};
+@@ -97,25 +104,25 @@
+ 	status = "okay";
+ 
+ 	partition@0 {
+-		label = "u-boot";
+-		reg = <0x0000000 0x100000>;
++		label = "uboot";
++		reg = <0x0000000 0xe0000>;
+		read-only;
+	};
+
+-	partition@100000 {
+-		label = "uImage";
+-		reg = <0x0100000 0x400000>;
++	partition@0xe0000 {
```

Comment:
Ah, and the syntax is partition@hexoffset, so not `partition@0xe0000` but `partition@e0000`
Please change that for all partitions and kernels.